provision/provision-ssh

```diff
@@ -10,6 +10,20 @@ scriptPath="provision/provision.sh"
 targetPath="/home/ubuntu/provision.sh"
 bastionIP=$(terraform output bastion_ip)
 
-scp -o StrictHostKeyChecking=no -i ${keyPath} $scriptPath ubuntu@$bastionIP:$targetPath
 
+#Wait until SSH on Bastion server is working
+echo "Attempting to SSH to Bastion server..."
+index=1
+set +x
+while (( $index != 0 ))
+do
+  scp -o StrictHostKeyChecking=no -i ${keyPath} $scriptPath ubuntu@$bastionIP:$targetPath
+  case $? in
+    (0) echo "${index}> Success"; break ;;
+    (*) echo "${index}> Bastion SSH server not ready yet..." ;;
+  esac
+  sleep 5
+  ((index+=1))
```

Comment:
This is code I originally got from Wayne - the point was to simply increment how many times the scp was attempted before successful.  We could simply change it to 

while (( $index <= 10 ))